sql/mob_pools.sql

```diff
@@ -1315,7 +1315,7 @@ INSERT INTO `mob_pools` VALUES (1254,'Eriri_Samariri','Eriri_Samariri',196,0x000
 INSERT INTO `mob_pools` VALUES (1255,'Erlik','Erlik',121,0x0000700100000000000000000000000000000000,4,4,12,240,100,0,1,0,0,1,0,0,193,1155,0,0,28,1,0,121);
 INSERT INTO `mob_pools` VALUES (1256,'Ermit_Imp','Ermit_Imp',165,0x0000BE0600000000000000000000000000000000,4,4,7,240,100,0,0,0,0,0,0,0,0,0,0,0,2,1,0,165);
 INSERT INTO `mob_pools` VALUES (1257,'Esbat','Esbat',46,0x0000000100000000000000000000000000000000,1,1,11,240,100,0,1,0,0,1,0,0,396,1667,0,0,0,1,0,46);
-INSERT INTO `mob_pools` VALUES (1258,'Escarp_Murex','Escarp_Murex',0,0x00005D0900000000000000000000000000000000,4,5,11,240,100,0,0,0,0,0,0,0,0,0,0,0,2,1,0,0);
+INSERT INTO `mob_pools` VALUES (1258,'Escarp_Murex','Escarp_Murex',187,0x00005D0900000000000000000000000000000000,4,5,11,240,100,0,0,0,0,0,0,0,0,0,0,0,2,1,0,187);
```

Comment:
note 2 self: should be 2 types each of murex/clinoid/limule/amoeba, diff elemental resistances and skills.